templates/otp.conf.j2

```diff
@@ -0,0 +1,9 @@
+description "start OpenTripPlanner process"
+
+start on {{ upstart_start_on }}
+stop on runlevel [!2345]
+
+chdir {{ otp_bin_dir }}
+script
+    exec sudo -u {{ otp_user }} ./otp -a --graphs {{ otp_data_dir }} --server
```

Comment:
Oh yeah, that example was from a machine serving the entire Northeast.  :)

It depends on the graph size.  Once loaded, the graph will take up a constant amount of space in memory, so it's easy enough to simply load the graph to see how big it is.  When it's initially loading, though, it will temporarily spike to consume a greater amount, so it's good to leave some headroom.

To generalize for a role, it might make sense to look at the size of the graph output from the build step, and multiply that by some constant.  The build step, however, will also need a variable amount of memory, depending primarily on the size of the OSM dataset for the covered area.  (Is the build using the `otp` script?  Probably should change that too.)

Memory size should probably just be a variable for the role, I guess.